addons/im_livechat/static/src/embed/common/thread_patch.xml

```diff
@@ -2,7 +2,7 @@
 <templates xml:space="preserve">
     <t t-inherit="mail.Thread" t-inherit-mode="extension">
         <xpath expr="//*[@name='content']" position="before">
-            <div t-if="props.thread?.livechatWelcomeMessage" class="bg-100 py-3">
+            <div t-if="props.thread?.livechatWelcomeMessage" class="bg-100 py-3" name="LivechatWelcomeMessage">
```

Comment:
`name` is not a valid attribute on `div` elements. 
https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Attributes
Consider `data-name` instead.

Reply:
Or a specific class name.